### PR TITLE
feat: Initialise nighthawk load generator interface

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

File: helpers/load_test_interface.go [hunk 1]
```diff
@@ -168,6 +169,90 @@ func WRK2LoadTest(opts *models.LoadTestOptions) (map[string]interface{}, *period
...
+		err = errors.Wrap(err, "error while unmarshaling data to the FortioHTTPRunner")
+		logrus.Error(err)
+		return nil, nil, err
+	}
+
+	resultsMap := map[string]interface{}{}
+	err = json.Unmarshal(bd, &resultsMap)
+
+	if err != nil {
+		err = errors.Wrap(err, "error while unmarshaling data to map")
```
Comment (leecalcote) [inline helpers/load_test_interface.go:247]: Are these errors displayed as toaster notifications? If so, please cleanup.

**Comment by kushthedude** [comment]
These are logs and displayed on stdout, this PR is for server side changes.

File: models/load-test.go [hunk 2]
```diff
@@ -21,6 +21,9 @@ const (
 
 	// Wrk2LG - represents the wrk2 load generator
 	Wrk2LG LoadGenerator = "wrk2"
+
+	// NighthawkLG - represents the nighthawk load generator
+	NighthawkLG LoadGenerator = "nighthawk"
```
Comment (leecalcote) [inline models/load-test.go:26]: Performance preferences for this load generator are currently the same as for the other two load generators, right?

Comment (kushthedude) [reply models/load-test.go:26]: I didn't get this, but if it is about the preferences set up from settings then yes they will be same for each load generator.

File: helpers/load_test_interface.go (hunk 1 above)
Comment (kushthedude) [reply helpers/load_test_interface.go:247]: These are logs and displayed on stdout, this PR is for server side changes.

File: models/load-test.go (hunk 2 above)
Comment (leecalcote) [reply models/load-test.go:26]: Roger. SGTM

**Pull request merged**